inspirehep/modules/deposit/dojson/fields/literature.py

```diff
@@ -221,7 +221,7 @@ def languages(self, key, value):
     if value not in ('en', 'oth'):
         return [unicode(dict(languages).get(value))]
     else:
-        return [value]
+        return []
```

Comment:
> Jan, all submissions seem to get (ln en) behind the title.
Neutrino Factory and Muon Collider Collaboration R&D Activities (In en)
https://inspirehep.net/record/1427157
-- Annette